docs/guides/development/nodejs/nodejs-twitter-bot/index.md

```diff
@@ -0,0 +1,294 @@
+
+```javascript
+	function pressSelect() {
+
+	    var weeklyReplyText = soundtrackArray[soundtrackArrayElement] + " Here's your soundtrack for the week!";
+	    T.post('statuses/update', { status: weeklyReplyText }, gameOver2);
+
+	    function gameOver2(err, reply) {
+	        if (err) {
+	            console.log(err.message);
+	            console.log("Game Over");
+	        } else {
+	            console.log('Tweeted: ' + reply.text);
+	        }
+	    }
+	}
+
+	 const job1 = schedule.scheduleJob(rule, pressSelect);
+
+	 job1.on("Every Day Tweet", pressSelect);
+```
+</br>
+
+Here is the function used to tweet on a schedule, which I’ve named `pressSelect`. `pressSelect` has the `replyText` variable, slightly changed to be an original tweet rather than a comment on a different tweet, but pulls from the same array. The `gameOver` function is also present, though renamed just to be safe. Since `gameOver` and `gameOver2` are local variables within their respective functions, there shouldn’t be any issues. However, I'm just being safe.
+
+The final part of this code is creating a variable called `job1`. `job1` is the `scheduleJob` method from the node-schedule object at the top of the code. I pass in `rule` created earlier in the code and `pressSelect` as parameters. We then use an event listener with `pressSelect` passed in again. That's it for the code.
+
+## Running the Code
+
+To test your code and ensure it works, type `npm run develop` into the terminal. If you get a Twit error about consumer keys, ensure there are no spaces between the variable, equals sign, and key itself in your .env file. If the error persists, you may have copied your keys wrong. You can always generate them again and copy them directly into the .env file. If you’d like to test `pressSelect` on its own and make sure it works, you can just comment out the last two lines of the code and call `pressSelect ` directly. Under the commented out code, just type
+
+```
+
+	pressSelect();
+
+```
+and the function will run. This way, you don’t have to wait for whatever day and hour you scheduled node-schedule for.
+
+Once the bot is running, if you want to test the way it responds to other users, log in to another Twitter account and tweet at your bot. You should be able to see some action in the terminal that tells you it’s working, followed by the response on Twitter.
+
+## Hosting Your Code
```

Comment:
- Made the sections after the `Host the Code` section into subsections/H3s to group them together under this topic
- Made a `Create a Linode` section https://deploy-preview-4492--nostalgic-ptolemy-b01ab8.netlify.app/docs/guides/nodejs-twitter-bot/#create-a-linode
- Made a new `Log in and Secure the Server` section
  https://deploy-preview-4492--nostalgic-ptolemy-b01ab8.netlify.app/docs/guides/nodejs-twitter-bot/#log-in-and-secure-the-server
  The previous instructions said that making a non-root-user wasn't mandatory, because there's no other important services running on the Linode, which I understand. But, I think we should make it mandatory, because a compromised server can act as a malicious actor in a botnet, attacking other servers